nifi-nar-bundles/nifi-cdc/nifi-cdc-mysql-bundle/nifi-cdc-mysql-processors/src/main/java/org/apache/nifi/cdc/mysql/processors/CaptureChangeMySQL.java

```diff
@@ -269,6 +284,31 @@ public class CaptureChangeMySQL extends AbstractSessionFactoryProcessor {
             .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
             .build();
 
+    public static final PropertyDescriptor EVENTS_PER_FLOWFILE_STRATEGY = new PropertyDescriptor.Builder()
+            .name("capture-change-mysql-events-per-flowfile")
+            .displayName("Events Per FlowFile Strategy")
+            .description("Specifies the strategy to use when writing events to FlowFile(s)")
+            .required(true)
+            .sensitive(false)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .allowableValues(N_EVENTS_PER_FLOWFILE_STRATEGY, ONE_TRANSACTION_PER_FLOWFILE_STRATEGY)
+            .defaultValue(N_EVENTS_PER_FLOWFILE_STRATEGY.getValue())
+            .expressionLanguageSupported(ExpressionLanguageScope.NONE)
+            .build();
+
+    public static final PropertyDescriptor NUMBER_OF_EVENTS_PER_FLOWFILE = new PropertyDescriptor.Builder()
+            .name("capture-change-mysql-number-of-events-per-flowfile")
+            .displayName("Number of Events Per FlowFile")
```

Comment:
This seems more verbose than necessary, what do you think about shorter names?
```suggestion
            .name("events-per-flowfile")
            .displayName("Events Per FlowFile")
```